### PR TITLE
it's almost an app now

### PR DESCRIPTION
**Pull request by spioupiou**
Please do not get scared by the amount of files, those are all icons with different sizes to fit all possible devices :)

I followed the below documentation:
https://kitt.lewagon.com/knowledge/tutorials/from_web_app_to_pwa

Please find a video on how to add the app to your homescreen on iOS (for iPhones, the 'add to homescreen' option is only available on safari, other browsers don't allow it 😢 ) There seems to be a workaround with an app called shortcuts according to SO (link below) if you really don't want to use safari:

https://stackoverflow.com/questions/50319831/can-i-use-add-to-home-screen-in-chrome-on-an-ios-device

For android: it seems that you can add the website to your homescreen pretty easily with Chrome or Firefox! See bottom of le wagon documentation for more info. 

Screen Shot 2022-03-09 at 20 09 16

As I will join the instance via QR code, having the app on my homescreen doesn't really make sense for me. But it could be a cool addition for Master Pitcher @carlnoval!

https://user-images.githubusercontent.com/68413600/157429888-29ae023d-7e61-4ecb-be6a-6f00c37eafdd.mov

### CODE REVIEW

File: app/assets/config/manifest.json.erb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+    <% files = Dir.entries(Rails.root.join("app/assets/images/icons/")).select {|f| !File.directory? f
+    } %>
+    <% files.each_with_index do |file, index| %>
+      <% match = file.match(/.+-(?<size>\d{
+      2,
+      3
+    }x\d{
+      2,
+      3
+    }).png/) %>
```
Comment (carlnoval) [inline app/assets/config/manifest.json.erb:16]: Any reason why this regex is on multi-line? from the tutorial it just looks like this:
```erb
<% files.each_with_index do |file, index| %>
      <% match = file.match(/.+-(?<size>\d{2,3}x\d{2,3}).png/) %>
      {
        "src":
```

**Comment by carlnoval** [comment]
@spioupiou so I just have one comment. Everything else looks ok. I was also able to easily save to homepage with chrome on android.

Yeah apple bitches a lot with their safari browser, can't install it on windows/android OS last I checked.

File: app/assets/config/manifest.json.erb (hunk 1 above)
Comment (spioupiou) [reply app/assets/config/manifest.json.erb:16]: whaaat I have no idea how it ended up like that. Will fix right away!

Comment (spioupiou) [reply app/assets/config/manifest.json.erb]: Done :)

**Comment by carlnoval** [comment]
Looks good will merge now.

**Pull request merged**